src/CSS/Article.css

```diff
@@ -0,0 +1,44 @@
+.article {
+    border: solid black 2px;
+    margin: 20px;
+    padding: 20px;
+}
+
+
+
+#title {
+    grid-area: title;
+    margin: 1rem;
+}
+
+#date {
+    grid-area: date;
+    margin: 1rem
+}
+
+#author {
+    grid-area: author;
+    margin: 1rem
+}
+
+#topic {
+    grid-area: topic;
+    margin: 1rem
+}
+
+#votes {
+    grid-area: votes;
+    margin: 1rem
+}
+
+#comments {
+    grid-area: comments;
+    margin: 1rem
+}
+
+#img {
```

Comment:
Your id's are a bit generic here and as they need to be unique I'd recommend using a naming convention like BEM to keep them unique. 

ie article__img as there's going to be lot's of other images around